app/containers/ItunesContainer/reducer.js

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ * ItunesContainer reducer
+ *
+ */
+import produce from 'immer';
+import { createActions } from 'reduxsauce';
+import get from 'lodash/get';
+
+export const initialState = {
+  searchTerm: null,
+  gridData: [],
+  searchesError: null
```

Comment:
searchError would be more appropriate

Reply:
Understood, searchError is better var name, will change it

Reply:
Updated in the PR